s/showtime/stone.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : showtime
+version     : 48.1
+release     : 1
+homepage    : https://apps.gnome.org/Showtime/
+upstreams   :
+    - https://download.gnome.org/sources/showtime/48/showtime-48.1.tar.xz : 1b22202dbe540440797d5a065030c29c242b210e3d4ec5ceb42e320ecc544ab9
+summary     : Watch without distraction
+description : |
+    Play your favorite movies and video files without hassle. Showtime features simple playback controls that fade out of your way when you're watching, fullscreen, adjustable playback speed, multiple language and subtitle tracks, and screenshots — everything you need for a straightforward viewing experience.
+license     : GPL-3.0-or-later
+builddeps   :
+    - binary(msgfmt)
+    - binary(update-desktop-database)
+    - pkgconfig(blueprint-compiler)
+    - pkgconfig(gtk4)
+    - pkgconfig(libadwaita-1)
+rundeps     :
+    - gst-plugins-rs
```

Comment:
I don't think this is right. It's correct on Solus because we don't break up the gst-plugins-rs package, but on Aeryn we split out the gtk4 gstreamer plugin which is the only one this package actually depends on. Also since this is a python program the autodep isn't picking up any of the other required deps so it needs those specified as well.

Something like this I imagine:
```
rundeps:
  - binary(python3
  - gstreamer-plugin-gtk4
  - gtk-4
  - libadwaita
```
And then we probably want it to pull in `gstreamer-plugins-good` and `gstreamer-plugins-bad` at the very least so that it gets some codec support.